src/py/flwr/__init__.py

```diff
@@ -22,3 +23,8 @@
     "server",
     "simulation",
 ]
+
+try:
+    __version__ = version(__name__)
+except:
+    pass
```

Comment:
@sisco0 what is the reason for the try/except? Can't we just use `__version__ = version(__name__)`? Are there cases in which this would fail?